templates/components/form/form.jsx

```diff
@@ -56,7 +75,8 @@ var Form = React.createClass({
     }
 
     return (
-     <label className={className} key={id} htmlFor={id}>
+     <label ref={id+'Label'} className={this.getInLabelClasses(id)} key={id} htmlFor={id}>
```

Comment:
I don't know what this name means. Can we make it `getLabelClasses`, since that's what it does?